lib/active_model_serializers/test/schema.rb

```diff
@@ -0,0 +1,77 @@
+require 'json_schema'
+
+module ActiveModelSerializers
+  module Test
+    module Schema
+      def assert_response_schema(schema_path = nil, message = nil)
```

Comment:
would you mind adding comments to the effect of what nil schema_path/message mean and what types of data they are? Yardoc or pseudo-yardoc is preferred.

```ruby
# A Minitest Assertion that the response is valid per the schema
# which uses the given error message
# @params schema_path [String] blah nil blah else
# @params message [String] blah nil blah else
# @example
#   get :something
#   assert_response_schema
```

and the matcher just returns bool?